includes/User.php

```diff
@@ -246,6 +247,11 @@ class User {
 	static $idCacheByName = array();
 
 	/**
+		* @var string the service auth token (currently helios); should NEVER be cached
+	 */
+	private $globalAuthToken = null;
```

Comment:
The caching should only be for [these variables](https://github.com/Wikia/app/blob/dev/includes/User.php#L99). The construction of the cache object happens [here](https://github.com/Wikia/app/blob/dev/includes/User.php#L434). So, unless there is some other magic this variable should not be cached.